packages/services/src/event/index.ts

```diff
@@ -165,5 +165,18 @@ export namespace Event {
   /**
    * The interface for the event bus front-end.
    */
-  export interface IManager extends EventManager {}
+  export interface IManager extends IDisposable {
+    /**
+     * The server settings used to make API requests.
+     */
+    readonly serverSettings: ServerConnection.ISettings;
+    /**
+     * An event stream that emits and yields each new event.
+     */
+    stream: Event.Stream;
```

Comment:
Would you mind making this one read only?

```suggestion
    readonly stream: Event.Stream;
```

Reply:
Thanks Fred!